src-cpp/KafkaConsumerImpl.cpp

```diff
@@ -251,6 +251,7 @@ RdKafka::KafkaConsumerImpl::close () {
   while (rd_kafka_outq_len(rk_) > 0)
     rd_kafka_poll(rk_, 10);
   rd_kafka_destroy(rk_);
+  rk_ = NULL;
```

Comment:
I'm guessing this will mean that any method call after `close` will segfault, and I also guess that maybe you get better failure behavior in this case if `rk_` weren't set to NULL.

Reply:
Yeah, that's not good, changed it so it only calls close(), the destroy is done in the dtor as it is supposed.
Otherwise we would need to protect each object `rk_` access with a mutex, which would be costly and also add unnecessary concurrent constraints.

Thanks for being so switched on!